packages/cli/src/commands/push.ts

```diff
@@ -89,9 +94,9 @@ async function runActionIfRequested(externalOptions: any): Promise<void> {
   return action(options);
 }
 
-async function runActionIfNeeded(contractName: string, network: string, options: any): Promise<void> {
+async function runActionIfNeeded(contractAlias: string, network: string, options: any): Promise<void> {
```

Comment:
This `contractName` was not even used, right?

Reply:
Correct.